pkg/prefetch/prefetch.go

```diff
@@ -0,0 +1,33 @@
+package prefetch
+
+import (
+	"encoding/json"
+	"sync"
+
+	"github.com/containerd/containerd/log"
+)
+
+var (
+	prefetchMsg MessagePrefetch
```

Comment:
I'm not sure, but I think this global variable may cause some inconsistencies. For instance, if two pods are started, the first one needs to pull a larger image than the second one. Can they get the correct prefetch files? Do you have a test for this case?